aws_cloudwatch.go

```diff
@@ -264,6 +264,7 @@ func (iface cloudwatchInterface) getMetricData(filter *cloudwatch.GetMetricDataI
 
 func createStaticDimensions(dimensions []dimension) (output []*cloudwatch.Dimension) {
 	for _, d := range dimensions {
+		d := d
 		output = append(output, &cloudwatch.Dimension{
```

Comment:
You have to copy the iteration variable since its value changes as you iterate through the slice...this isn't visible in single-dimensional test